modules/nf-core/genomad/endtoend/main.nf

```diff
@@ -0,0 +1,123 @@
+process GENOMAD_ENDTOEND {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::genomad=1.5.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/genomad:1.5.2--pyhdfd78af_0':
+        'biocontainers/genomad:1.5.2--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path  genomad_db
+    val   score_calibration
+
+    output:
+    tuple val(meta), path("*_aggregated_classification/*_aggregated_classification.tsv")    , emit: aggregated_classification
+    tuple val(meta), path("*_annotate/*_taxonomy.tsv")                                      , emit: taxonomy
+    tuple val(meta), path("*_find_proviruses/*_provirus.tsv")                               , emit: provirus
+    tuple val(meta), path("*_score_calibration/*_compositions.tsv")                         , emit: compositions , optional: true
```

Comment:
Harshil alignment is not right on the , 😀

Reply:
as you are correcting the harshil alignment, could you also get rid of the extra empty lines in the stub sections @CarsonJM Thanks